pkg/server/service.go

```diff
@@ -25,12 +25,12 @@ import (
 
 	"github.com/containerd/containerd"
 	"github.com/containerd/containerd/plugin"
+	"github.com/containerd/cri/pkg/streaming"
```

Comment:
will be difficult to keep in sync.

Reply:
Would it be easier to keep in sync if it was in a separate repository (and some script to automate extracting / diffing with the upstream k/k whenever it tags a release ?)